product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/index.mdx

```diff
@@ -36,7 +36,7 @@ You can also modify your cluster by installing Postgres extensions. See [Postgre
    | Retention period for backups                         | Additional Settings | &mdash;                                                                                                                                                                                                                                                                                                                                                                   |
    | Read-only workloads                                  | Additional Settings | Enabling read-only workloads can incur higher cloud infrastructure charges.                                                                                                                                                                                                                                                                                          |
    | Identity and Access Management (IAM) Authentication  | Additional Settings | Turn on the ability to log in to Postgres using AWS IAM credentials. You must then run a command to add each user’s credentials to a role that uses IAM authentication in Postgres. See [IAM authentication for Postgres](../01_postgres_access/#iam-authentication-for-postgres).                                                                                   |
-
+   | Maintenance                                          | Additional Settings | The 60-minute maintenance window specifies the start time, not necessarily the end time, of maintenance events.                                                                                  |
```

Comment:
Change the setting to `Custom maintenance window` and place it between Read only workloads and IAM authentication to match with the UI.